src/templates/tests/unit/widgets/HelloWorld.ts

```diff
@@ -1,13 +1,12 @@
-import * as registerSuite from 'intern/lib/interfaces/object';
+const { registerSuite } = intern.getInterface('object');
 import harness from '@dojo/test-extras/harness';
 
 import { v } from '@dojo/widget-core/d';
 
 import HelloWorld from '../../../src/widgets/HelloWorld';
 import * as css from '../../../src/widgets/styles/HelloWorld.m.css';
 
-registerSuite({
-	name: 'HelloWorld',
+registerSuite('HelloWorld', {
 	'render'() {
```

Comment:
would we want to wrap this in `tests: {}` to make it easier to add other lifecycle type things?

Reply:
I’m a fan of the bdd test interfaces (which means you don’t need that `tests` wrapper...)